### PR TITLE
Addresses case where adding a file didn't start the file

### PR DESCRIPTION
**Pull request by bikeath1337**
Addresses #276. Files weren't starting after a canceled file if the canceled file had not started and there was nothing left to upload when that file was canceled. The fix was to removed the non-started file from the queuedFiles list.

### CODE REVIEW

File: evaporate.js [hunk 1]
```diff
@@ -14,7 +14,7 @@
 
 /***************************************************************************************************
  *                                                                                                 *
- *  version 2.0                                                                                    *
+ *  version 2.1                                                                                    *
```
Comment (cvn) [inline evaporate.js]: What about using 2.0.1? Semver says bug fixes should increment the patch version.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,10 @@
+# v2.0.1
+
+## Bug Fixes ##
+- Issue #277. It was possible for a queued file to not start if the previous file was canceled
+  before it had started. Also addressed edge cases and race conditions when canceling a list
+  of files that have been submitted. Thanks to @cvnfor the test cases.
```
Comment (cvn) [inline CHANGELOG.md]: Thanks for the mention! There's a missing space between my username and "for".
